.github/workflows/build-test-cxx.yml

```
@@ -54,6 +54,11 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```

For safety reasons let's use a tagged version.
```suggestion
      uses: jlumbroso/free-disk-space@v1.3.1
```